packages/storybook-helpers/viewports.ts

```diff
@@ -0,0 +1,83 @@
+export const VIEWPORT = {
+  PHONE: "phone",
+  P_TABLET: "pTablet",
+  L_TABLET: "lTablet",
+  LAPTOP: "laptop",
+  DESKTOP: "desktop",
+} as const;
+
+// eslint-disable-next-line import/no-unused-modules
+export type Viewport = (typeof VIEWPORT)[keyof typeof VIEWPORT];
+
+type Dimension = {
+  width: number;
+  height: number;
+};
+
+type DimensionTuple = [number, number]; // [width, height];
+
+export const DIMENSIONS: Record<Viewport, Dimension> = {
```

Comment:
Could you leave a comment here and in the hydrogen config to help us remember to keep them in sync?

Reply:
Hopefully this makes sense 😅 

[add comment for width dimensions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8478/commits/3b1666fe8b56955ff7bfd5ed6c5f51195f8cb4e6)